docs/connector-development/config-based/tutorial/4-reading-data.md

```diff
@@ -26,10 +26,10 @@ See the [catalog guide](https://docs.airbyte.com/understanding-airbyte/beginners
 
 Let's define the stream schema in `source-exchange-rates-tutorial/source_exchange_rates_tutorial/schemas/rates.json`
 
-You can download the JSON file describing the output schema with all currencies [here](https://raw.githubusercontent.com/airbytehq/airbyte/master/airbyte-cdk/python/docs/tutorials/http_api_source_assets/exchange_rates.json) for convenience and place it in `schemas/`.
+You can download the JSON file describing the output schema with all currencies [here](../../tutorials/cdk-tutorial-python-http/exchange_rates_schema.json) for convenience and place it in `schemas/`.
```

Comment:
point to this file docs/connector-development/tutorials/cdk-tutorial-python-http/exchange_rates_schema.json